examples/example-provider-authentication/terraform.tfvars.example

```diff
@@ -0,0 +1,7 @@
+# -------------------------------------------------------------------------------------------
```

Comment:
as mentioned in the credential provider args

Reply:
will consider this for the data repo, I added with-tfvars" to the example folder name to be clear with the example that is shown here.